seahorn/trusty/user/base/lib/tipc/proof/tipc_send1/main.c

```diff
@@ -0,0 +1,35 @@
+#include <seahorn/seahorn.h>
+#include <lib/tipc/tipc.h>
+#include <trusty_ipc.h>
+#include <stdlib.h>
+
+extern handle_t nd_handle(void);
+extern int nd_size_t(void);
+//extern void* can_fail_malloc(int);
+extern int msg_send_called;
+
+/** Test harness for tpic_send1 */
+int main(void) {
+
+ // msg_is_send_called = 0;
+
+  /* assumptions */
+ msg_send_called = 0;
+ handle_t handle = nd_handle();
+
+ size_t buf_len = nd_size_t();
+ //void *buf = can_fail_malloc(buf_len);
+ void *buf = malloc(buf_len);
```

Comment:
need to use our allocation library. This has potentially undefined behaviour. The buffer is allocated but is never written to. It is then given to a function that might read from it.

Reply:
ok

Reply:
one way to fix this locally is to call memhavoc after malloc. @danblitzhou can help with that.